models/User.ts

```diff
@@ -300,6 +302,13 @@ export default function (
     return groupIds.includes(groupId) === true;
   };
 
+  User.prototype.isGroupAdmin = async function (
+      groupId: number
+  ): Promise<boolean> {
+    const groupIds = await this.getGroups();
+    return groupIds.some(({id, GroupUsers}) => id === groupId && GroupUsers.admin === true);
```

Comment:
should this be `groupUser`?

Reply:
Not according to sequelize I don't think.